### ISSUE TITLE
actor security

### ISSUE THREAD

**Issue by despiegk** [threefoldtech/jumpscaleX_core#256]
- admin user = threebot user, only he can go to actors by default
- need to use the session object to make sure only admin (own user) can access
- you can have multiple admin (so suggest to use the circle:admin for that purpose)

# remark for actor methods

- or method is 100% public (need to specify, maybe a decorator?)
- or default is method only accessible by admins
- or in actor method we deal with checking the circles membership

edit after reopen:

TODO:
- [ ] remove authentication header from actor
- [ ] remove authentication check from  gedis handler 
- [ ] update base crud actor to check for the required right in each method:
  - [ ] new > w
  - [ ] set > w
  - [ ] get > r
  - [ ] delete > d

**Comment by Pishoy** [comment]
after adding js_shell before ship2.scheduler.sleep_time, it works
![image](https://user-images.githubusercontent.com/21987477/69249154-199e9c80-0bb6-11ea-85e6-a35376875e9f.png)

- jsx version
```
3BOTDEVEL:jsx:jumpscaleX_core: git log -1
commit 0f70acd4e4a630c46d0d48e16fe9bd40fddec741 (HEAD -> development_3git, origin/development_3git)
Author: kristof de spiegeleer <>
Date:   Wed Nov 20 06:08:26 2019 +0100

    don't remove servers from j.... yet

```

**Comment by andrewayoub** [comment]
This issue is reopened because the specs are changed.

TODO:
- [ ] remove authentication header from actor
- [ ] remove authentication check from  gedis handler 
- [ ] update base crud actor to check for the required right in each method:
  - [ ] new > w
  - [ ] set > w
  - [ ] get > r
  - [ ] delete > d

**Comment by andrewayoub** [comment]
this is blocked on https://github.com/threefoldtech/home/issues/377

**Comment by despiegk** [comment]
close for a new story, 3bot security
